src/cpp/src/llm_pipeline_static.hpp

```diff
@@ -10,6 +10,45 @@
 namespace ov {
 namespace genai {
 
+struct StaticLLMPipelineFactory {
```

Comment:
I think there could be a better namespace job done, clearly `statik::` could be a namespace here, so we'd get `statik::StatelessLLMPipeline` (the old class) and `statik::StatefulLLMPipeline` (the new one).

`statik::` is picked to avoid the clash with the keyword, it could be `Static::` too.

Reply:
Thank you!

Reply:
Fixed!

Reply:
@dmatveev @TolyaTalamanov , please help to disambiguate: we allow user to pass dynamic stateful OpenVINO model into our new pipeline, where we are hiding things like converting of model to static and making it stateless. Should we this way still name the pipeline as `static_llm::StatefulLLMPipeline`, as it still works with the static and stateless models inside? Or it can really be named as `Stateful` because now, `LLMCompiledModel` doesn't expose to user additional inputs and outputs, that correspond to states? (However, I don't know if by this logic the pipeline is still static)

Reply:
> I think there could be a better namespace job done, clearly `statik::` could be a namespace here, so we'd get `statik::StatelessLLMPipeline` (the old class) and `statik::StatefulLLMPipeline` (the new one).
> 
> `statik::` is picked to avoid the clash with the keyword, it could be `Static::` too.

Why not just `npu_impl`?

Reply:
@AsyaPronina The new pipeline is definitely `stateful` as pipeline doesn't handle dynamism and states any longer. Nobody cares what is inside